linux_os/guide/system/logging/journald/service_systemd-journal-remote_disabled/rule.yml

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+title: 'Disable systemd-journal-remote Socket'
+
+description: |-
+    Journald supports the ability to receive messages from remote hosts,
+    thus acting as a log server. Clients should not receive data from
+    other hosts.
+    NOTE:
+        The same package, systemd-journal-remote , is used for both sending
+        logs to remote hosts and receiving incoming logs.
+        With regards to receiving logs, there are two services;
+        systemd-journal-remote.socket and systemd-journal-remote.service.
+
```

Comment:
I see that you are aiming at the "socket" unit.
I suggest using the following macro: https://github.com/ComplianceAsCode/content/blob/e227941887ea63aa060c81368d24ffe712e1367c/shared/macros/01-general.jinja#L289

Reply:
I was actually unsure about this, they only mention the socket, but could the service somehow enable the socket. I still haven't checked their unit files tbh